src/server.rs

```diff
@@ -452,6 +452,12 @@ async fn index(
 
     response.headers.extend(&global_response_headers);
 
+    if req.uri().path().eq("/docs") {
+        response
```

Comment:
I'd expect a more fundamental fix of response headers rather than checking path

Reply:
IMO there should be only one `Content-Type` header in the response, maybe it's better to 
- add some content-type resolution logic to prioritize `text/html`; or
- add an `exclude_paths: list[str]` argument to `set_response_header`; or
- add a `custom_response_header: dict[str, str]` argument to`add_route` 

Any of these is better than hard coding "/docs"

Reply:
@kigawas @sansyrox PTAL